test/puppetlabs/puppetdb/generative/spec.clj

```diff
@@ -0,0 +1,119 @@
+(comment
```

Comment:
It seems like we should remove this as there's no way for us to make sure the code still works, maybe it can be squirreled away in a branch?